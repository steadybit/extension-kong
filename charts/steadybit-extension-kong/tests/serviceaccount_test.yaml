templates:
  - serviceaccount.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: { }
