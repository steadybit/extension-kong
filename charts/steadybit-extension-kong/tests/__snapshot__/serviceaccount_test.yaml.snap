manifest should match snapshot:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      labels: null
      name: steadybit-extension-kong
      namespace: NAMESPACE
